packages/gatsby/cache-dir/loader.js

```diff
@@ -506,7 +506,7 @@ export class ProdLoader extends BaseLoader {
     super(loadComponent, matchPaths)
 
     if (pageData) {
-      this.pageDataDb.set(pageData.path, {
+      this.pageDataDb.set(findPath(pageData.path), {
```

Comment:
just for ref - this is how we fill pageData cache - https://github.com/gatsbyjs/gatsby/blob/fa70491bb6413797266d003e2fed7b680e964fd7/packages/gatsby/cache-dir/loader.js#L191-L202 and there we use `pagePath` as key which is result of `findPath(rawPath)`, and prewarming cache should use cache key produced with same method